packages/theme-patternfly-org/templates/mdx.css

```
@@ -294,6 +294,7 @@
 
 .ws-mdx-content {
   flex-grow: 1;
+  width: 100%;
```

This works great, though the root issue is that the `min-width` of a flex child is `min-content` - the generally recommended way to fix that is to set `min-width: 0;`. 

```suggestion
  min-width: 0;
```